tests/unit/support/assertRender.ts

```diff
@@ -216,6 +217,32 @@ registerSuite('support/assertRender', {
 					v('div', { }, [ v('span', {}, [ w(MockWidget, { foo: 'baz', baz() { } }) ]) ])
 				);
 			}, AssertionError, 'Render unexpected');
+		},
+
+		'compare unique classes only'() {
```

Comment:
Classes can be the following as well:

```
'foo' == [ 'foo' ] // should be true
'bar' == 'bar'  // should be true
'foo' == [ 'bar' ] // should be false
```

I was going to mention it in the logic above, but it made more sense to indicate the tests that should pass/fail here.

Reply:
Tests and logic updated